src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -64,4 +65,4 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyFileVersion("2.18.0.2986")]
+[assembly: AssemblyFileVersion("2.18.0.4447")]
```

Comment:
We usually skip the changes to this file, would you revert the changes to this file?